pydarn/plotting/fan.py

```diff
@@ -60,10 +69,15 @@ def __str__(self):
     @classmethod
     def plot_fan(cls, dmap_data: List[dict], ax=None,
                  scan_index: Union[int, dt.datetime] = 1,
-                 parameter: str = 'v', cmap: str = None,
-                 groundscatter: bool = False, zmin: int = None,
-                 zmax: int = None, colorbar: bool = True,
-                 colorbar_label: str = '', title: bool = True,
+                 ranges: List = [], boundary: bool = True,
+                 line_color: str = 'black', alpha: int = 0.5,
+                 parameter: str = 'v', lowlat: int = 30, cmap: str = None,
```

Comment:
@carleyjmartin I have just fixed the lowlat issue let me know if it works on your end thanks.

Reply:
It's working well for northern hemisphere, but it appears to be being ignored for southern hemisphere plots, but I can't seem to find why that would be.


I'm also still concerned about the kwargs situation. A lot of the variables on the end of the function set up here (and in plot_fov) are not used in this function but are read through to be used in other functions if required (lowlat I believe was one of them), so we use the **kwargs here for that instead of defining them here as well. Are all of these added key words used within the plot_fan function explicitly? I think this might also be the reason we're getting a few pytests fail due to some kwargs being red in and some forgotten (i.e. pytest is failing on line 651 in fan.py due to 'alpha' not being brought through to be used in kwargs?) The upgrade to using kwargs to pass through variables to inner functions was introduced in a recent version, so it could also be that you were working from a previous version of this file. I'm not sure. Let me know if you need more help on this, I think it's more of a problem incorporating the new code into the existing code base and layout rather than anything being wrong with it. Marina is currently working on getting cartopy involved before any plotting, so we will probably need to tweak a few things for that too - I think we're going to get out a patch release for v2.2(.1?) and then we can think about merging this and Marina's PR into develop for v3.0 early next year.